### PR TITLE
Graph view feature

### PR DESCRIPTION
**Pull request by brascooriol**
Adding graph view feature

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -32,3 +32,4 @@ colorama~=0.4
 paginate~=0.5
 regex>=2022.4
 requests~=2.26
+mkdocs-obsidian-interactive-graph-plugin
```
Comment (rbrasco) [inline requirements.txt]: Add package version

Comment (brascooriol) [reply requirements.txt]: done

Comment (brascooriol) [reply requirements.txt]: done

**Review by rbrasco — APPROVED** [review]
LGTM

**Pull request merged**